druid/examples/processing_on_other_threads.rs

```diff
@@ -0,0 +1,102 @@
+#![windows_subsystem = "windows"]
```

Comment:
First I've seen this. Guessing it's something related to getting to work on Windows?

Reply:
It suppresses the console window. I think without being conditional, this would damage portability. I think it's orthogonal to the concern about blocking.

Reply:
Without this, if you run the executable in windows, it pops up a console window. It's indeed orthogonal, happy to remove it if you prefer.

Reply:
Let's remove it. Yes the console pops up, as it does with every other example. We'll eventually deal with it more generally.

Reply:
Cool, I've removed it.